index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="https://use.typekit.net/wng7kgg.css">
```

Comment:
Add file style approved

Reply:
- font type: briller - approved for header -CG